### PR TITLE
Change cloudhub configuration access method from Get() to global variable 'Config'

### PR DESCRIPTION
**Pull request by kuramal**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test

/kind feature

**What this PR does / why we need it**:
Change cloudhub configuration access method from Get() to global variable 'Config'

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cloud/pkg/cloudhub/cloudhub.go [hunk 1]
```diff
@@ -65,10 +65,10 @@ func (a *cloudHub) Start() {
 
 	servers.StartCloudHub(messageq)
 
-	if hubconfig.Get().UnixSocket.Enable {
+	if hubconfig.CONFIG.UnixSocket.Enable {
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/cloudhub.go]: It's better to use Config here?

Comment (kuramal) [reply cloud/pkg/cloudhub/cloudhub.go]: that's good

**Comment by kuramal** [comment]
ref https://github.com/kubeedge/kubeedge/pull/1464

**Pull request closed without merging**